iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -45,12 +45,12 @@ <h1 class="footnote-mode-only" data-i18n="inspector.footnoteProperties">Footnote
         </div>
       </div>
       <div class="controls">
-        <div class="outline-button">
+        <div class="outline-button" title="Document outline">
```

Comment:
Out of curiosity, is it somehow possible to do i18n for `title`?

Reply:
I did look into this at the time, but obviously didn't look hard enough as it turns out it's very easy.   Done in a4d4a35.

I'm looking at some tools (i18next-parser) to help make the translation files maintainable, but will tackle that in a separate PR.